x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_stats.ts

```diff
@@ -68,8 +68,7 @@ export const getStatsFromFindingsEvaluationsAggs = (
   const failedFindings = findingsEvaluationsAggs.failed_findings.doc_count || 0;
   const passedFindings = findingsEvaluationsAggs.passed_findings.doc_count || 0;
   const totalFindings = failedFindings + passedFindings;
-  if (!totalFindings) throw new Error("couldn't calculate posture score");
-  const postureScore = calculatePostureScore(passedFindings, failedFindings);
+  const postureScore = calculatePostureScore(passedFindings, failedFindings) || 0;
```

Comment:
we used to throw if there were no findings, now we assume it is 0. is this intentional?

Reply:
yes 0 is now the expected result 👍